Sources/StreamChatUI/UIConfig+ChannelList.swift

```diff
@@ -13,7 +13,7 @@ public extension _UIConfig {
         public var channelListItemView: _ChatChannelListItemView<ExtraData>.Type = _ChatChannelListItemView<ExtraData>.self
         public var channelViewCell: _ChatChannelListCollectionViewCell<ExtraData>.Type =
             _ChatChannelListCollectionViewCell<ExtraData>.self
-        public var newChannelButton: ChatChannelCreateNewButton<ExtraData>.Type = ChatChannelCreateNewButton<ExtraData>.self
+        public var newChannelButton: _ChatChannelCreateNewButton<ExtraData>.Type = _ChatChannelCreateNewButton<ExtraData>.self
```

Comment:
Just a reminder that you should document this as well.

Reply:
✅ Done!

Reply:
Also added on more test, for UIConfig changes if you want to take a look at it 👀

Reply:
Same here, this can also be just `UIButton`